tests/subsys/bluetooth/fast_pair/storage/account_key_storage/src/test_corrupted_data.c

```diff
@@ -28,7 +28,7 @@ static uint8_t next_account_key_id(uint8_t account_key_id)
 	return account_key_id + 1;
```

Comment:
The function should not be used for backends with Owner Account Key and multiple Account Keys support (and it still would be called for these backends)

Reply:
We should fix it or add a build assert `(ACCOUNT_KEY_CNT == 1) || !IS_ENABLED(CONFIG_TEST_BT_FAST_PAIR_STORAGE_OWNER_ACCOUNT_KEY)` (it should be ok too since all of our backends pass this check)

Reply:
I agree with assert approach for now we can make follow up ticket to fix it later.